scripts/teamDataScript.js

```diff
@@ -0,0 +1,617 @@
+        }
+    });
+
+    var objName = pitAnswerObjName(season,event,team);
+    localStorage.setItem(objName,JSON.stringify(all_team_QA_pairs_obj));
+    
+    $('#pitTableConfirmation').text('Answers saved.');
+    showAllPitDataTable();
+}
+
+function showAllPitDataTable() //shows pit stats of all teams for the current event in one table
+{
+    var season = getCookie('currCheckedSeason');
+    var event = getCurrEvent();
+    var cname = season + ' pitQuestions';
+    var questionsArr = getCookie(cname).split('Ω');
+    $('#allTeamsPitAnswers').html('');
+    $('#allPitAnswersTitle').text(`All Pit Scouting Data for the '${event} event'.`);
+
+    var tableHeader = '<tr><th></th>';
+    for (var i = 0; i < questionsArr.length-1; i++)
+    {
+
+        tableHeader += `<th>${questionsArr[i]}</th>`;
+    }
+    tableHeader += '</tr>'; //end of row
+    $('#allTeamsPitAnswers').html(tableHeader); //default start of table
+
+    var keys = Object.keys(localStorage);
+    for (var j = 0; j < keys.length; j++) //iterate through localStorage objects
+    {
+        if (!keys[j].endsWith('pit')) continue;
+
+        var team = keys[j].split('/')[3]; //based on structure of pitStorageObjName
+        var newRowHTML = `<tr><td>${team}</td>`;
+        var pitObjValue = JSON.parse(localStorage.getItem(keys[j])); //value is an object with more objects inside
+        var innerKeys = Object.keys(pitObjValue);
+
+        for (var k = 0; k < innerKeys.length; k++) //starts at 1 because first value is blank, it reads the table header
+        {
+            var pair = pitObjValue[k];
+            var key = Object.keys(pair)[0];
+            var answer = pair[key];
+            newRowHTML += `<td>${answer}</td>`;
+        }
+        //newRowHTML += '</tr>' //end of row
+
+        $('#allTeamsPitAnswers').append(newRowHTML);
+    }
+}
+
+
+
+//Match Stuff
+function loadMatch()
+{
+    $('#currMatchHeader').text('');
+    var season = getCookie('currCheckedSeason');
+    var event = getCurrEvent();
+    var teamNum = getCurrTeamNumber();
+
+    if (teamNum.trim().length == 0)
+    {
+        $('#matchTableConfirmation').text('No Team selected.');
+        $('#matchNumberStuff').hide();
+        return;
+    }
+    $('#matchNumberStuff').show();
+    loadMatchNumberList();
+
+    var tableHeader = '<tr> <th><b>Element</b></th> <th><b>Frequency</b></th> </tr>';
+    $('#matchQuestionTable').html(tableHeader); //default start of table
+
+    //retrieve elements for current season
+    var cname = `/season_config/${season}/` //starter bit of season_config_name
+    var cookieList = document.cookie.split(';');
+
+    for (var i = 0; i < cookieList.length; i++)
+    {
+        var currCName = cookieList[i].trim().split('=')[0];
+
+        if (!(currCName.startsWith(cname))) continue;
+
+        var element = currCName.substring(currCName.lastIndexOf('/')+1);
+        var value = getCookie(currCName);
+
+        var inputHTML = `<input type="text" class="q-input">`;
```

Comment:
we should setup where things (which will not be all) that we want to track that are numbers are `type="number"` to allow for up/down buttons or the numeric input.